src/ImageService.Data/ImageMessageRepository.cs

```diff
@@ -48,7 +48,7 @@ public bool Remove(List<Guid> imageIds)
       foreach (Guid imageId in imageIds)
       {
         _provider.ExecuteRawSql($@"DELETE FROM {DbImageMessage.TableName} WHERE Id = '{imageId}' OR ParentId = '{imageId}' OR
-            Id IN (SELECT ParentId FROM ImagesProjects WHERE Id = '{imageId}' AND ParentId IS NOT NULL);");
+          Id IN (SELECT ParentId FROM ImagesProjects WHERE Id = '{imageId}' AND ParentId IS NOT NULL);");
```

Comment:
change ImagesProjects to {DbImageMessage.TableName}

Reply:
done